pkg/scheduler/util/utils_test.go

```diff
@@ -122,54 +126,96 @@ func TestMoreImportantPod(t *testing.T) {
 	}
 }
 
-func TestRemoveNominatedNodeName(t *testing.T) {
+func TestClearNominatedNodeName(t *testing.T) {
+	statusErr := &apierrors.StatusError{
+		ErrStatus: metav1.Status{Reason: metav1.StatusReasonUnknown},
```

Comment:
nits: `Message` field is used when this status error is formatted and logged.

```suggestion
		ErrStatus: metav1.Status{Reason: metav1.StatusReasonUnknown, Message: "status error"},
```